lib/usd/ui/cmds/USDImportDialogCmd.cpp

```diff
@@ -67,45 +70,65 @@ MStatus USDImportDialogCmd::doIt(const MArgList& args)
 {
 	MStatus st;
 	MArgParser argData(syntax(), args, &st);
+	if (!st) return st;
 
-	if(argData.isFlagSet(kClearDataFlag))
+	if (argData.isQuery())
```

Comment:
Note: this diff looks bad, but all I really did was add the isQuery() section and then put the code below into the else (so it all got indented).

Reply:
Yeah good suggestion, I do that.